test/core/storage/leveldb/leveldb_integration_test.cpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "testutil/storage/base_leveldb_test.hpp"
+
+#include <array>
+#include <exception>
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include "storage/leveldb/leveldb.hpp"
+#include "storage/leveldb/leveldb_error.hpp"
+#include "testutil/outcome.hpp"
+
+using namespace kagome::storage;
+namespace fs = boost::filesystem;
+
+struct LevelDB_Integration_Test : public test::BaseLevelDB_Test {
+  LevelDB_Integration_Test()
+      : test::BaseLevelDB_Test("/tmp/kagome_leveldb_integration_test") {}
```

Comment:
can we create this file during SetUp of the test and remove during TearDown?

Reply:
this is how it works already